docs/api/structures/ipc-renderer-event.md

```diff
@@ -1,9 +1,6 @@
 # IpcRendererEvent Object extends `Event`
 
 * `sender` [IpcRenderer](../ipc-renderer.md) - The `IpcRenderer` instance that emitted the event originally
-* `senderId` Integer - The `webContents.id` that sent the message, you can call `event.sender.sendTo(event.senderId, ...)` to reply to the message, see [ipcRenderer.sendTo][ipc-renderer-sendto] for more information. This only applies to messages sent from a different renderer. Messages sent directly from the main process set `event.senderId` to `0`.
-* `senderIsMainFrame` boolean (optional) - Whether the message sent via [ipcRenderer.sendTo][ipc-renderer-sendto] was sent by the main frame. This is relevant when `nodeIntegrationInSubFrames` is enabled in the originating `webContents`.
```

Comment:
It's logical that these would be removed as part of this change, but they were not called out as deprecated in #39091. I think a good litmus test should be we're not deleting anything not marked with `_Deprecated_`, which we'd fail in this case.

Can we do a follow-up PR to mark these with `_Deprecated_` and call it out in `docs/breaking-changes.md`?

Reply:
https://github.com/electron/electron/pull/39381